legistar/base.py

```diff
@@ -139,6 +165,8 @@ def parseDetails(self, detail_div):
                 for link in field_2.xpath('.//a'):
                     value.append({'label': link.text_content().strip(),
                                   'url': self._get_link_address(link)})
+            elif key == 'eComment':
```

Comment:
a way that would be pretty clean to factor this out to the event class is to have this block look like 

```python
            if field_2.find('.//a') is not None:
                value = []
                for link in field_2.xpath('.//a'):
                    value.append({'label': link.text_content().strip(),
                                  'url': self._get_link_address(link)})
            elif 'href' in field_2.attrib:
                value = {'label': field_2.text_content().strip(),
                         'url': self._get_link_address(field_2)}
            elif self._detail_parse(field):
                value = self._detail_parse(field)
            else:
                value = field_2.text_content().strip()
```

and define `_detail_parse` to return None on the base class, and override on the event class.